### PR TITLE
Python: Verify support for Python 3.13

### PR DESCRIPTION
**Pull request by amotl**
## About
[Python 3.13.0](https://www.python.org/downloads/release/python-3130/) has been released on Oct. 7, 2024. This PR intends to add CI verification.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -135,6 +135,7 @@ def run(self):
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
```
Comment (amotl) [inline setup.py:138]: Python 3.13 deprecated stdlib's [cgi](https://docs.python.org/3.11/library/cgi.html) module. [apistar](https://github.com/encode/apistar) uses it, but it ceased being maintained.
```python
import apistar
```
```python
ModuleNotFoundError: No module named 'cgi'
```

Comment (amotl) [reply setup.py:138]: apistar is exclusively used at this spot:

https://github.com/kennethreitz/responder/blob/f90955a9b9a201a5de21028795c32a8c0d2687df/responder/ext/schema/__init__.py#L128-L136

Do you see a way to replace that using a different library? Please advise.

/cc @taoufik07, @tomchristie, @tabotkevin

**Comment by tabotkevin** [comment]
@amotl  Yes, I already replaced API star on Responder before `https://github.com/kennethreitz/responder/pull/480`

**Comment by amotl** [comment]
Thanks, @tabotkevin. Apparently, the ingredients of GH-480 have been reverted down the line, despite your PR appears to be have been merged correctly?

**Comment by kennethreitz** [comment]
@amotl i ended up reverting a whole series of changes to try to get the repo closer to the state it was when i stopped actively maintaining it. It's in a much better spot for me now — but we may need to replay a few PRs that are valuable.

**Comment by amotl** [comment]
@tabotkevin: Your patch GH-480 has been converged into GH-543, and submitted again.

**Review by tabotkevin — APPROVED** [review]
LGTM!

**Pull request merged**